pkg/controller/installation/installation_controller.go

```diff
@@ -160,37 +186,37 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 		}
 	}
 
-	//UPDATE STATUS
+	// UPDATE STATUS
 	err = r.client.Status().Update(r.context, instance)
 	if err != nil {
 		// The 'Update' function can error if the resource has been updated by another process and the versions are not correct.
 		if k8serr.IsConflict(err) {
 			// If there is a conflict, requeue the resource and retry Update
-			log.Info("Error updating Installation resource status. Requeue and retry.")
+			logrus.Info("Error updating Installation resource status. Requeue and retry.", err)
 			return reconcile.Result{
 				Requeue:      true,
-				RequeueAfter: time.Second * 10,
+				RequeueAfter: time.Second,
```

Comment:
Just curious, what was the reasoning for reducing this to a second? Should the other 10 second requeue on L189 also be reduced?

Reply:
No reason, was just testing and forgot to change it back! Thanks